tests/Options/SearchTablesOptionsTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Keboola\Test\Options;
+
+use Keboola\StorageApi\Options\SearchTablesOptions;
+use Keboola\Test\StorageApiTestCase;
+
+class SearchTablesOptionsTest extends StorageApiTestCase
+{
+    public function testGetDefaults()
+    {
+        $options = new SearchTablesOptions;
+        $this->assertSame([
+            'metadataKey' => null,
+            'metadataValue' => null,
+            'metadataProvider' => null,
+        ], $options->toArray());
+    }
+
+    public function testCreate()
+    {
+        $options = new SearchTablesOptions;
+        $this->assertInstanceOf(SearchTablesOptions::class, $options);
```

Comment:
to by zajímalo, kdy tohle neprojde :thinking:

není to už skoro `$this->assertTrue(true);` ? :-)

Reply:
Jo to tam ostalo, protože tam předtým bylo `SearchTablesOptions::create()` takže tam to bylo na místě.

Reply:
neni true jako true https://thedailywtf.com/articles/What_Is_Truth_0x3f_